orc/src/main/java/com/netflix/iceberg/orc/ORC.java

```diff
@@ -74,7 +72,7 @@ public WriteBuilder schema(Schema schema) {
       return this;
     }
 
-    public OrcFileAppender build() {
```

Comment:
Does `OrcFileAppender` have any methods that are specific to the ORC implementation that callers may need to use?

Reply:
There's `OrcFileAppender#getSchema` which returns the underlying ORC schema as a `TypeDescription` . 

Also, now that I think about it. This could be problematic .  `OrcFileAppender` uses `VectorizedRowBatch` as a concrete type when implementing `FileAppender<T>` .  Just returning  a `FileAppender`, as I've done here, may not be a valid change in this specific scenario.
In that regard, ORC's `FileAppender` is deviating from Avro's and Parquet's where `FileAppender`'s record type is parameterized. Which implies we cannot use Iceberg's own record type when writing ORC data. 

ORCFileAppender seems to be breaking an abstraction boundary here when it is exposing an internal class [VectorizedRowBatch] to Iceberg's main APIs.